TypeCobol.Test/Misc/TestPictureParsing.cs

```diff
@@ -21,35 +23,58 @@ public class TestPictureParsing {
 			TestConversion("XXXXXXXXX",     DataType.Alphanumeric);
 			TestConversion("xxxxxxxxx",     DataType.Alphanumeric);
 			TestConversion("****.**",       DataType.NumericEdited);
+			TestConversion("****,**",       DataType.NumericEdited, false);
```

Comment:
`false` is the default value so this is equivalent to `TestConversion("****,**", DataType.NumericEdited);`
Didn't you mean `TestConversion("****,**", DataType.NumericEdited, true);` instead ?

Reply:
Yes it was a mistake.
Note that PictureValidator only check that the character used for decimal point appear only once. The other character (comma or dot) can appear multiple times.